package/MDAnalysis/__init__.py

```diff
@@ -167,18 +167,18 @@
 
 # Registry of Readers, Parsers and Writers known to MDAnalysis
 # Metaclass magic fills these as classes are declared.
-_READERS = {}
-_READER_HINTS = {}
-_SINGLEFRAME_WRITERS = {}
-_MULTIFRAME_WRITERS = {}
-_PARSERS = {}
-_PARSER_HINTS = {}
-_SELECTION_WRITERS = {}
-_CONVERTERS = {}
+_READERS = {}  # type: dict
```

Comment:
Is there a reason for the different style of annotation? If not, I'd rather have the style consistent:

```python
_READERS: Dict = {}
```

Also, the annotation can be more specific:

```python
_READERS: Dict[str, ProtoReader] = {}
```

Reply:
Ok I'll make the changes

Reply:
I made the changes like you suggested . But the tests are failing

Reply:
I looked at it and it looks like the issue is caused by the imports in `__init__.py`.

I tried two ways of fixing the problem:
* I guarded the import with `TYPE_CHECKING`, but then I needed to reference the type with forward references (using the name of the types as strings instead of just providing the types)
* providing the types in a stub file (see <https://mypy.readthedocs.io/en/stable/stubs.html>)

Both methods seem to work in the sense that nowhere mypy complains about not having annotations for these variables. However, I know the annotations are wrong, yet I do not get any errors about them. So I would suggest keeping it simple for now and just annotating them as `Dict`. It will not be as precise as it should, but at least it won't be wrong.